templates/jfleg/templates.yaml

```diff
@@ -0,0 +1,100 @@
+dataset: jfleg
+templates:
+  13ada088-1e08-4c18-8042-50da516cd8b2: !Template
+    id: 13ada088-1e08-4c18-8042-50da516cd8b2
+    jinja: "{{sentence}}\n\nThe above sentence is hard to read.\n\nFour possible correct\
+      \ options are:\n\n|||\n\n{{ corrections[0]}} \n\nor\n\n{{ corrections[1]}} \n\
+      \nor\n\n{{ corrections[2]}}\n\nor\n\n{{ corrections[3]}} "
+    name: template_v2
+    reference: ''
+    task_template: true
```

Comment:
i would remove this one
1/ it doesn't much stronger signal compared to v1
2/ you have to deal with possibly repetitive options
<img width="1272" alt="Capture d’écran 2021-06-14 à 3 33 35 PM" src="https://user-images.githubusercontent.com/16107619/121949046-09b15500-cd26-11eb-87bd-7d7fabdd8897.png">